packages/@react-spectrum/breadcrumbs/src/Breadcrumbs.tsx

```diff
@@ -9,14 +9,13 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-
 import {ActionButton} from '@react-spectrum/button';
 import {BreadcrumbItem} from './BreadcrumbItem';
-import {classNames, useDOMRef, useStyleProps} from '@react-spectrum/utils';
+import {classNames, useDOMRef, useResizeObserver, useStyleProps} from '@react-spectrum/utils';
```

Comment:
I think a merge got messed up, these changes are already merged into main

Reply:
Owh! I will resolve this ASAP, thank you!